corehq/apps/api/tests/test_messaging_event_api.py

```diff
@@ -600,3 +583,33 @@ def _create_sms_messages(domain, count, randomize):
         sms, _ = create_fake_sms(domain, randomize=randomize)
         results.append(sms)
     return results
+
+
+def _serialized_messaging_event(sms):
+    return {
+        "id": sms.messaging_subevent_id,
+        "content_type": "sms",
+        "date": "2016-01-01T12:00:00",
+        "date_last_activity": sms.date_modified.isoformat() if sms.date_modified else "2016-01-01T12:00:00",
+        "case_id": None,
+        "domain": "qwerty",
+        "error": None,
+        "form": None,
+        'messages': [
+            {
+                'message_id': sms.id,
+                'backend': 'fake-backend-id',
+                'phone_number': '99912345678',
+                'content': 'test sms text',
+                'date': '2016-01-01T12:00:00',
+                'date_modified': sms.date_modified.isoformat() if sms.date_modified else None,
+                'direction': 'outgoing',
+                'status': 'sent',
+                'type': 'sms'
+            }
+        ],
+        # "id": 1,  # ids are explicitly removed from comparison
```

Comment:
should this be deleted now that `id` is included?

Reply:
ah, yes, thanks
```suggestion
```